testnet/stacks-node/Cargo.toml

```diff
@@ -49,6 +49,7 @@ tracing = "0.1.37"
 tracing-subscriber = { version = "0.3.17", features = ["env-filter"] }
 wsts = {workspace = true}
 mutants = "0.0.3"
+tiny_http = "0.12.0"
```

Comment:
Can you also remove `async-h1` and `async-std`?

Reply:
Not yet, because they're still used in bitcoin_regtest_controller.

Reply:
Although, maybe I could make those changes in this PR as well... I'll take a look.